# Get supported camera settings

---

CameraParam[] params
